Tests/SwiftDocCTests/Test Bundles/MixedLanguageFramework.docc/symbol-graphs/clang/MixedLanguageFramework.symbols.json

```diff
@@ -0,0 +1,545 @@
+{
+  "metadata" : {
+    "formatVersion" : {
+      "major" : 0,
+      "minor" : 5,
+      "patch" : 3
+    },
+    "generator" : "SymbolKit"
```

Comment:
Nit: this should probably be `clang`

Reply:
0b865e406ab029038d47aed5b0cea5be76a28bfb